src/contractTest/java/uk/gov/hmcts/reform/sscs/ContractTestDataProvider.java

```diff
@@ -33,43 +33,57 @@
 
-    protected static final String SERVICE_AUTHORIZATION = "ServiceAuthorization";
-    protected static final String IDAM_OAUTH2_TOKEN = "pact-test-idam-token";
-    protected static final String UNAUTHORISED_IDAM_OAUTH2_TOKEN = "unauthorised-pact-test-idam-token";
-    protected static final String SERVICE_AUTHORIZATION_TOKEN = "pact-test-s2s-token";
-    protected static final String UNAUTHORISED_SERVICE_AUTHORIZATION_TOKEN = "unauthorised-pact-test-s2s-token";
+    public static final String SERVICE_AUTHORIZATION = "ServiceAuthorization";
+    public static final String IDAM_OAUTH2_TOKEN = "pact-test-idam-token";
+    public static final String UNAUTHORISED_IDAM_OAUTH2_TOKEN = "unauthorised-pact-test-idam-token";
+    public static final String SERVICE_AUTHORIZATION_TOKEN = "pact-test-s2s-token";
+    public static final String UNAUTHORISED_SERVICE_AUTHORIZATION_TOKEN = "unauthorised-pact-test-s2s-token";
 
     public static final String MSG_200_HEARING = "Success (with content)";
-    public static final String MSG_400_HEARING = "Invalid hearing state for DELETE";
+    public static final String MSG_400_HEARING = "Invalid request";
     public static final String MSG_401_HEARING = "Unauthorised request";
     public static final String MSG_403_HEARING = "Forbidden request";
     public static final String MSG_404_HEARING = "Not Found request";
 
-    protected static final String CONTENT_TYPE = "Content-Type";
-    protected static final String APPLICATION_JSON = "application/json";
 
-    protected static final String HEARING_PATH = "/hearing";
-    protected static final String FIELD_STATUS = "status";
-    protected static final String BAD_REQUEST = "BAD_REQUEST";
-    protected static final String FIELD_MESSAGE = "message";
-    protected static final String FIELD_ERRORS = "errors";
-    protected static final int ZERO_LENGTH = 0;
-    protected static final Number ZERO_NUMBER_LENGTH = 0;
 
-    protected static final Map<String, String> authorisedHeaders = Map.of(
+    public static final String CONTENT_TYPE = "Content-Type";
+    public static final String APPLICATION_JSON = "application/json";
+
+    public static final String HEARING_PATH = "/hearing";
+    public static final String FIELD_STATUS = "status";
+    public static final String BAD_REQUEST = "BAD_REQUEST";
+    public static final String FIELD_MESSAGE = "message";
+    public static final String FIELD_ERRORS = "errors";
+    public static final int ZERO_LENGTH = 0;
+    public static final Number ZERO_NUMBER_LENGTH = 0;
+    public static final String FIELD_ID = "id";
+    public static final String VALID_CASE_ID = "123";
+    public static final String FORBIDDEN_CASE_ID = "456";
+    public static final String NOT_FOUND_CASE_ID = "789";
+
+    public static final String HEARING_RESPONSE_STATUS = "HEARING_REQUESTED";
+    public static final String HEARING_DATE = "2030-08-20T12:40";
+
+    private ContractTestDataProvider() {
+
```

Comment:
Will this later be filled? If not could we use a @NoArgsConstructor instead?

Reply:
Actually Utility classes should not have a public or default constructor , It needs to have defined private and even if we @NoArgsConstructor use build will fail checkstyle.